packages/features/bookings/Booker/Booker.tsx

```diff
@@ -115,6 +115,7 @@ const BookerComponent = ({
     columnViewExtraDays.current =
       Math.abs(dayjs(selectedDate).diff(availableSlots[availableSlots.length - 2], "day")) + addonDays;
   const prefetchNextMonth =
```

Comment:
This change was made here: https://github.com/calcom/cal.com/pull/9815/files#diff-d8cdd443610a4483db45dcc4da6a02bcff302ad6c591aea6fa1ee8646a52aedbR116

We only need `prefetchNextMonth` for the COLUMN_VIEW. The error `The requested time range is too long.` can still happen in the column view if the timespan shown is too long

Reply:
I think we can fix it for longer ranges completely by breaking down the range into multiple acceptable ranges and sending multiple requests? 

I saw someone did similar for Zoho Calendar

Reply:
Yes, I will make a follow-up PR 👍